intranet/static/js/common.nav.js

```diff
@@ -0,0 +1,39 @@
+$(function() {
+    var initX = null, initY = null, listening = false;
+    $(window).on("touchstart", function(e) {
+        initX = e.originalEvent.touches[0].clientX;
+        initY = e.originalEvent.touches[0].clientY;
+        listening = true;
+    });
+    $(window).on("touchend", function(e) {
+        listening = false;
+    });
+    $(window).on("touchmove", function(e) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 6.](https://www.codacy.com/app/tjcsl/ion/file/2286322831/issues/source?bid=3176160&fileBranchId=3189731#l11)